src/main/java/com/exasol/dbbuilder/dialects/exasol/VirtualSchema.java

```diff
@@ -55,6 +57,17 @@ private void addReservedProperties(final Builder builder) {
         }
     }
 
+    private void addDebugProperties() {
+        final String debugAddress = System.getProperty(DEBUG_ADDRESS);
+        if (debugAddress == null) {
+            return;
+        }
+        final String logLevel = System.getProperty(DEBUG_LOG_LEVEL);
```

Comment:
Strictly speaking, setting the debug level without the output address could be useful to. There is a database parameter called `SCRIPT_OUTPUT_ADDRESS`. This is a similar mechanism, but works deeper down in the database and catches more output in case something goes wrong. Especially at a stage where the actual VS is not yet ready.

See https://docs.exasol.com/db/latest/database_concepts/udf_scripts/debug_udf_script_output.htm?Highlight=script_output

Reply:
Thanks, I will change implementation and update user_guide.